vir/defs/high/operations_internal/position/statement.rs

```diff
@@ -84,3 +88,31 @@ impl Positioned for LeakAll {
         Default::default()
     }
 }
+
+impl Positioned for NewLft {
+    fn position(&self) -> Position {
+        self.position
+        // Default::default()
+    }
+}
+
+impl Positioned for EndLft {
+    fn position(&self) -> Position {
+        self.position
+        // Default::default()
+    }
+}
+
+impl Positioned for GhostAssignment {
+    fn position(&self) -> Position {
+        // self.position
```

Comment:
Why commented out? I think it should return `self.position` instead of `Default::default()`. This is also the reason for the panic. (Also below.)

Reply:
indeed! done.